### PR TITLE
bug/MOBILE-3610 Merchant address needs to be more flexible with the type of lat and lng

### PR DESCRIPTION
**Pull request by agustk**
Making the data type of latitude and longitude more flexible.

### CODE REVIEW

**Review by arturdm — CHANGES_REQUESTED** [review]

File: sdk/src/main/java/com/meniga/sdk/models/merchants/MenigaMerchantLocation.java [hunk 1]
```diff
@@ -14,8 +14,8 @@
 	protected String city;
 	protected String country;
 	protected String countryCode;
-	protected Double latitude;
-	protected Double longitude;
+	protected Object latitude;
```
[the remark this answers is not in the document]
Comment (agustk) [reply sdk/src/main/java/com/meniga/sdk/models/merchants/MenigaMerchantLocation.java]: Done

File: sdk/src/main/java/com/meniga/sdk/models/merchants/MenigaMerchantLocation.java [hunk 2]
```diff
@@ -48,14 +48,28 @@ public String getCountryCode() {
 	 * @return The merchant's latitude.
 	 */
 	public Double getLatitude() {
-		return latitude;
+		if (latitude == null) {
```
[the remark this answers is not in the document]
Comment (agustk) [reply sdk/src/main/java/com/meniga/sdk/models/merchants/MenigaMerchantLocation.java]: Good idea

[the remark this answers is not in the document]
Comment (agustk) [reply sdk/src/main/java/com/meniga/sdk/models/merchants/MenigaMerchantLocation.java]: Done

[the remark this answers is not in the document]
Comment (arturdm) [reply sdk/src/main/java/com/meniga/sdk/models/merchants/MenigaMerchantLocation.java]: Just two `@Nullable`s there and we're ready to go.

**Review by arturdm — APPROVED** [review]

**Pull request merged**